src/AutoBalance.cpp

```diff
@@ -432,16 +461,21 @@ std::map<uint32, uint32> LoadDistanceCheckOverrides(std::string dungeonIdString)
 }
 
 
-bool isEnabledDungeon(uint32 dungeonId)
+bool isDungeonInDisabledDungeonIds(uint32 dungeonId)
 {
-    return (enabledDungeonIds.find(dungeonId) != enabledDungeonIds.end());
+    return (std::find(disabledDungeonIds.begin(), disabledDungeonIds.end(), dungeonId) != disabledDungeonIds.end());
 }
 
-bool perDungeonScalingEnabled()
+bool isDungeonInMinPlayerMap(uint32 dungeonId, bool isHeroic)
 {
-    return (!enabledDungeonIds.empty());
+    if (isHeroic == true) {
+        return (minPlayersPerHeroicDungeonIdMap.find(dungeonId) != minPlayersPerHeroicDungeonIdMap.end());
+    } else {
+        return (minPlayersPerDungeonIdMap.find(dungeonId) != minPlayersPerDungeonIdMap.end());
+    }
```

Comment:
```suggestion
    if (isHeroic) {
        return (minPlayersPerHeroicDungeonIdMap.find(dungeonId) != minPlayersPerHeroicDungeonIdMap.end());
    }

    return (minPlayersPerDungeonIdMap.find(dungeonId) != minPlayersPerDungeonIdMap.end());
```

Reply:
I'm not particularly bothered by the change, but the net effect is the same, so help education me - is there a best practice or style guide item that explains why it should be done this way?

Reply:
I think I'm going to keep the if/else, but I'll drop the ` == true`. Thank you for the feedback!

Reply:
ok 👍